Add ubuntu to integration test

trombik opened the pull request.

Commit:
add ubuntu pair to integration

Commit:
rename

mitsururike commented on tests/integration/ubuntu_ubuntu/spec/default_spec.rb:
```diff
@@ -0,0 +1,35 @@
+    it "creates a file, /exports/rw/bar" do
+      current_server.ssh_exec("touch /exports/rw/bar")
+      r = current_server.ssh_exec("ls /exports/rw/bar")
+      expect(r).to match(%r{\/exports\/rw\/bar})
```
escape is not required inside %r{}

Commit:
remove forward slashes

https://github.com/reallyenglish/ansible-role-nfs-server/pull/7#discussion_r121059157

Commit:
support ada(4)

fixes build

Commit:
try different subnet to see if it fix build
